variables.tf

```diff
@@ -30,7 +30,13 @@ variable "kms_key_id" {
 variable "postfix" {
   type        = bool
   default     = true
-  description = "Postfix the user and policy names with Account and Policy"
+  description = "Postfix the user, policy and group names with Account, Policy and Group"
+}
+
+variable "group_names" {
+  type = set(string)
+  default = []
+  description = "Set of group names to attach to the user"
```

Comment:
Ow, and please keep the variables ordered alphabetically. So this should go between the `create_policy` and `policy` vars.